codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/client/FluentClientDecorator.kt

```diff
@@ -493,13 +504,34 @@ class FluentClientGenerator(
                     val inputType = symbolProvider.toSymbol(operation.inputShape(model))
                     val outputType = symbolProvider.toSymbol(operation.outputShape(model))
                     val errorType = operation.errorSymbol(model, symbolProvider, CodegenTarget.CLIENT)
+                    // TODO figure out how to properly get the RetryPolicy here
```

Comment:
Do you think it would be possible to use an opaque type here to hide the generics and still keep the customization in Smithy rather than in the SDK?